yass/main.go

```diff
@@ -18,7 +17,7 @@ const (
 
 func main() {
 	// Ensure site dir exists
-	if _, err := os.Stat(siteDirPath); errors.Is(err, os.ErrNotExist) {
+	if _, err := os.Stat(siteDirPath); os.IsNotExist(err) {
```

Comment:
I did find that function, but the docs for `os.IsNotExist()` recommended using `errors.Is` instead.

Reply:
Interesting! That’s news to me.